### ISSUE TITLE
Issue With Uuid Model ID

### ISSUE THREAD

**Issue by reasecret** [cviebrock/eloquent-taggable#139]
I'm using uuid for my model's id column and taggable_id column type is BIGINT. There is a conflict with column types.

Maybe this option can be change on "taggable.php" config file like this:

`'uuid' => true,
`

And this config could be check on migration file:

```
if (config('taggable.uuid')) {
     $table->uuidMorphs('taggable_id');
} else {
     $table->unsignedBigInteger('taggable_id');
}
```
This is just a suggestion.

**Comment by cviebrock** [comment]
Good idea @reasecret !  If you want to put together a PR for this feature, I'd be happy to add it quickly.
